### DISCUSSION TITLE
Taking over an existing formula

### DISCUSSION THREAD

**Discussion by seebeen** [Homebrew/discussions#4891] [Writing Formulae/Casks]
### Output of `brew config`

```shell
none
```

### Output of `brew doctor`

```shell
none
```

### Description of issue

Formula **nanorc** is not actively maintained. I've tried reaching out to its author numerous times.
If I fork it, improve it and release it - is there a process in place to replace the existing formula on brew?

**Comment by SMillerDev** [comment]
If your repo becomes the de-facto source for nanorc you can request the formula points to your repo, we need some pretty convincing arguments (usage numbers, other package managers switching or blessing from upstream) for that to be the case though.
